### ISSUE TITLE
micronaut-http-server-netty-3.4.3.jar reported as Trojan by Windows Defender

### ISSUE THREAD

**Issue by RZetko** [micronaut-projects/micronaut-core#7312]
### Expected Behavior

Windows Defender should not report micronaut-http-server-netty-3.4.3.jar as virus.

### Actual Behaviour

Windows Defender reports micronaut-http-server-netty-3.4.3.jar as Trojan when running first installation of packages using mvnw mn:run. Downgrading to previous version 3.4.2 seems to workaround this issue. Attached screenshot shows issue in detail. I also attached demo project downloaded from Micronaut Launch where it occured for me.

![Screenshot (109)](https://user-images.githubusercontent.com/6697764/166934713-338092dc-55ae-4e66-bfda-5209cdb57b8a.png)
 

### Steps To Reproduce

1. Create template application using Micronaut Launch with following:

> Micronaut 3.4.3
> Java 17
> Maven
> Junit
 
2. Unpack project
3. Run mvnw mn:run
4. Windows Defender warning notification should popup informing you of micronaut-http-server-netty-3.4.3.jar being detected as Trojan

[demo.zip](https://github.com/micronaut-projects/micronaut-core/files/8631647/demo.zip)

### Environment Information

- Operating system: Windows 11 x64, build 21H2 22000.613
- OpenJDK 17 x64, Microsoft build

### Version

3.4.3

**Comment by yawkat** [comment]
I can't see any security warning on that jar: https://www.virustotal.com/gui/file/b6922655ce60cb4b913a8212325e3e9ee15c9a83caedd6820bf022a104e2944b?nocache=1

Can you confirm that the file has the same hash (sha-256  b6922655ce60cb4b913a8212325e3e9ee15c9a83caedd6820bf022a104e2944b) on your system?

**Comment by RZetko** [comment]
Can confirm that sha-256 is same after uploading it to https://emn178.github.io/online-tools/sha256_checksum.html and calculating hash. Hash is b6922655ce60cb4b913a8212325e3e9ee15c9a83caedd6820bf022a104e2944b and I also attached screenshot.

I noticed after deleting .m2 directory and redownloading all packages on different 3.4.3 project that even micronaut-http-server-3.4.3.jar is reported as Trojan (also visible in screenshot). sha-256 for that file is a7e3cd29ffbceff1b8077873359625b777e8449cc76ae5015da63ac45c274085.


![Screenshot (110)](https://user-images.githubusercontent.com/6697764/167112026-6668a31e-6a65-437d-88d0-4e4316937add.png)
